src/Database/Migrations/2022-12-13-200557_add_ban_columns.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodeIgniter\Shield\Database\Migrations;
+
+use CodeIgniter\Database\Migration;
+
+class AddBanColumns extends Migration
+{
+    public function up(): void
+    {
+        // Users Table
+        $fields = [
+            'banned'      => ['type' => 'tinyint', 'after' => 'active', 'constraint' => 1, 'null' => 0, 'default' => 0],
+            'ban_message' => ['type' => 'varchar', 'after' => 'banned', 'constraint' => 255, 'null' => true],
+        ];
+        
+        $this->forge->addColumn('users', $fields);
```

Comment:
I think we can get rid of the `status` and `status_message` columns now. We will need to ensure we let people know about that in the changelog, though.